requirements.txt

```diff
@@ -2,7 +2,7 @@ ConfigArgParse
 pandas
 seaborn
 tqdm
-scvi==0.6.7
+scvi-tools
```

Comment:
you might as well put now `scvi-tools>=0.9.0a2`, we can release an a3 soon with SOLO too

Reply:
Will do